packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -117,11 +117,20 @@ class IconLabelButton extends StatelessWidget {
         child: iconLabel,
       );
     }
+    var buttonStyle = Theme.of(context).outlinedButtonTheme.style;
+    if (!includeText(context, includeTextWidth)) {
+      buttonStyle = buttonStyle.copyWith(
+        padding: MaterialStateProperty.resolveWith<EdgeInsets>((_) {
+          return EdgeInsets.zero;
+        }),
+      );
+    }
```

Comment:
since this is used in a couple places, let's put it in theme.dart in a method

`ButtonStyle denseAwareOutlinedButtonStyle(BuildContext context, double includeTextWidth)`